.devcontainer/devcontainer.json

```
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+	"name": "Rust",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/rust:1-1-bullseye"
```

I'm not sure why it's not the default, but Debian Bookworm is the current stable version (Bullseye is oldstable)
```suggestion
	"image": "mcr.microsoft.com/devcontainers/rust:1-1-bookworm"
```